src/common/controllers/ads-controller.js

```diff
@@ -312,6 +316,26 @@ class AdsController extends FakeEventTarget implements IAdsController {
     });
   }
 
+  _handleLiveEvery(): void {
+    const liveConfigAdBreaks = [];
+    this._configAdBreaks.forEach(adBreak => {
+      const {every, ads} = adBreak;
+      if (this._player.duration && every) {
+        const position = this._player.currentTime + every;
+        liveConfigAdBreaks.push({
+          every,
+          position,
+          ads,
+          played: false,
+          loadedPromise: Promise.resolve()
+        });
+      }
+    });
+    if (liveConfigAdBreaks.length) {
+      this._configAdBreaks = liveConfigAdBreaks;
```

Comment:
so you override also the non-every breaks. in purpose?
whats about post roll (position -1)? I know it's live but we can get live ended in a particular use case.
can we push them instead?